### PR TITLE
Sb2k16 add profile

### PR DESCRIPTION
**Pull request by sb2k16**
### Description
Add bio and photo for sb2k16 for data prepper release 2.10.

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _community_members/sb2k16.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      url: '/community/members/souvik-bose.html'
+github: sb2k16
+job_title_and_company: 'Software Development Engineer at AWS'
+personas:
+  - author
+permalink: '/community/members/souvik-bose.html'
+redirect_from: '/authors/sb2k16/'
+---
+
+Souvik Bose is a Software Development Engineer at AWS working in search services. He is a contributor to the Data Prepper project.
```
Comment (natebower) [inline _community_members/sb2k16.md:24]:
```suggestion
Souvik Bose is a Software Development Engineer at AWS working on search services. He is a contributor to the Data Prepper project.
```

**Review by natebower — APPROVED** [review]
@sb2k16 LGTM with one minor change. Thanks! 

Cc: @dlvenable @pajuric

File: _community_members/sb2k16.md (hunk 1 above)
Comment (sb2k16) [reply _community_members/sb2k16.md]: Thanks @natebower. I have addressed this issue.

**Review by natebower — APPROVED** [review]
LGTM

**Review by nateynateynate — APPROVED** [review]
Thanks! LGTM.

**Pull request merged**